plugins/platforms/list.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 const joi = require('joi');
 const schema = require('screwdriver-data-schema');
-const listSchema = joi.array().items(schema.models.platform.get);
+const listSchema = joi.array().items(schema.models.platform.get).label('List of Platforms');
```

Comment:
No need to change this if you're going to merge https://github.com/screwdriver-cd/screwdriver/pull/52